be/src/exec/pipeline/pipeline_driver_dispatcher.cpp

```diff
@@ -8,11 +8,17 @@
 
 namespace starrocks::pipeline {
 
-GlobalDriverDispatcher::GlobalDriverDispatcher(std::unique_ptr<ThreadPool> thread_pool)
-        : _driver_queue(std::make_unique<DriverQueueWithWorkGroup>()),
-          _thread_pool(std::move(thread_pool)),
+GlobalDriverDispatcher::GlobalDriverDispatcher(std::unique_ptr<ThreadPool> thread_pool, bool is_use_workgroup)
+        : _thread_pool(std::move(thread_pool)),
           _blocked_driver_poller(new PipelineDriverPoller(_driver_queue.get())),
-          _exec_state_reporter(new ExecStateReporter()) {}
+          _exec_state_reporter(new ExecStateReporter()),
+          _is_use_workgroup(is_use_workgroup) {
+    if (_is_use_workgroup) {
+        _driver_queue = std::make_unique<QuerySharedDriverQueue>();
+    } else {
+        _driver_queue = std::make_unique<DriverQueueWithWorkGroup>();
+    }
```

Comment:
Maybe `_is_use_workgroup==true` should use `DriverQueueWithWorkGroup`, and `_is_use_workgroup==false` should use `QuerySharedDriverQueue`. Or maybe `_is_use_workgroup` should be renamed to `_is_realtime`.